### ISSUE TITLE
[Bug] The api of getProfile can not be accessed 

### ISSUE THREAD

**Issue by wangyao2016** [apache/incubator-hugegraph#1882]
### Bug Type (问题类型)

rest-api (结果不合预期)

### Before submit

- [X] 我已经确认现有的 [Issues](https://github.com/hugegraph/hugegraph/issues) 与 [FAQ](https://hugegraph.github.io/hugegraph-doc/guides/faq.html) 中没有相同 / 重复问题

### Environment (环境信息)

- Server Version: v0.13.0
- Backend: RocksDB x 1
- OS: 8 CPUs, 32 G RAM, Centos 7.6
- Data Size:  xx

### Expected & Actual behavior (期望与实际表现)

Expected
`
{
    "service": "hugegraph",
    "version": "0.13.0.0",
    "doc": "https://hugegraph.github.io/hugegraph-doc/",
    "api_doc": "https://hugegraph.github.io/hugegraph-doc/clients/hugegraph-api.html",
    "apis": [
        "auth",
        "filter",
        "graph",
        "gremlin",
        "job",
        "metrics",
        "profile",
        "raft",
        "schema",
        "traversers",
        "variables"
    ]
}
`

 Actual behavior 
`
{
    "exception": "class java.lang.IllegalArgumentException",
    "message": "Invalid request uri ''",
    "cause": ""
}
`

### Vertex/Edge example (问题点 / 边数据举例)

```javascript
I think the reason for the problem is: There is a problem accessing the route definition
```

**Comment by wangyao2016** [comment]
The pic of Actual behavior :
![image](https://user-images.githubusercontent.com/16890042/168511908-40ded10a-547a-4f1f-8a5f-4fe27bb9a92d.png)

And I have a pr for this. [fix bud: get profile api cannot be accessed](https://github.com/apache/incubator-hugegraph/pull/1883)

**Comment by javeme** [comment]
fixed by #1883 & #1893
